Add GQLstatus to `NoApplicableIndex` and `EagerOperator` (#148)
neo4j/docs-status-codes: modules/ROOT/pages/notifications/all-notifications.adoc

----
CREATE INDEX FOR (n:Person) ON (n.name)
----

======
[.include-with-GQLSTATUS-code]
======

Query::
+
[source, cypher, role="noplay"]
----
LOAD CSV FROM 'file:///ignore/ignore.csv' AS line WITH * MATCH (n:Person{name:line[0]}) RETURN line, n
----

Returned GQLSTATUS code::
03N93

Returned Status Description::
info: no applicable index.
`LOAD CSV` in combination with `MATCH` or `MERGE` on a label that does not have an index may result in long execution times.
Consider adding an index for label `Person`.

Suggestions for improvement::
Create an index on the label and property you match.
+
[source, cypher, role="noplay"]
----
CREATE INDEX FOR (n:Person) ON (n.name)
----
======
=====

[#_neo_clientnotification_statement_eageroperator]
=== EagerOperator

This notification is returned when the execution plan for a query contains the `Eager` operator.

==== Notification details

[.tabbed-example]
=====
[.include-with-neo4j-code]
======

[cols="<1s,<4"]
|===

> |Code

Author: ```suggestion
|Neo4j code
```

[#_neo_clientnotification_statement_noapplicableindex]
=== NoApplicableIndex

This notification is returned when using `LOAD CSV` with a `MATCH` or a `MERGE` clause that matches a non-indexed label.

> This will most likely not perform well on large data sets.

Author: ```suggestion
This may not perform well on large data sets.
```

> If you add an index, `LOAD CSV` together with a `MATCH` or a `MERGE` clause may speed up the query.

Reviewer: I think this make it sound like it is the `LOAD CSV` rather than the index which speeds up the query. Maybe the word ordering can be changed or the sentence slightly rewritten in some other way to make it clearer?

> If you add an index, `LOAD CSV` together with a `MATCH` or a `MERGE` clause may speed up the query.

Author: ```suggestion
Adding an index could improve the query speed.
```
